src/Kunstmaan/AdminBundle/Entity/Exception.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace Kunstmaan\AdminBundle\Entity;
+
+use DateTimeInterface;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+
+/**
+ * @ORM\Entity(repositoryClass="Kunstmaan\AdminBundle\Repository\ExceptionRepository")
+ * @ORM\Table(name="kuma_exception")
+ * @UniqueEntity("hash")
+ * @ORM\HasLifecycleCallbacks()
+ */
```

Comment:
I would add a regular index on the "mark" column.

Reply:
@treeleaf, ready.

Reply:
Thanks. Nice work, this PR!